modules/recommended-etcd-practices.adoc

```diff
@@ -16,7 +16,19 @@ For large and dense clusters, etcd can suffer from poor performance if the keysp
 
 For more information about defragmenting etcd, see the "Defragmenting etcd data" section.
 
-Because etcd writes data to disk and persists proposals on disk, its performance depends on disk performance. Slow disks and disk activity from other processes can cause long fsync latencies. Those latencies can cause etcd to miss heartbeats, not commit new proposals to the disk on time, and ultimately experience request timeouts and temporary leader loss. Run etcd on machines that are backed by SSD or NVMe disks with low latency and high throughput. Consider single-level cell (SLC) solid-state drives (SSDs), which provide 1 bit per memory cell, are durable and reliable, and are ideal for write-intensive workloads.
+Because etcd writes data to disk and persists proposals on disk, its performance depends on disk performance.
+Etcd requires a low latency block device for optimal performance and stability because the concensus protocol depends on persistently storing metadata to a log using Write-Ahead Logging (WAL). Slow disks and disk activity from other processes can cause long `fsync` latencies.
+
+Those latencies can cause etcd to miss heartbeats, not commit new proposals to the disk on time, and ultimately experience request timeouts and temporary leader loss. High write latencies also lead to an OpenShift API slowness, therefore the performance of the whole cluster would be affected.
+
+[IMPORTANT]
+====
+It is not recommended to co-locate other workloads in the Control Plane nodes.
+====
+
+It is recommended to run etcd on a block device able to write at least 50 IOPS of 8000 bytes in lengthy sequentially with a latency of 20ms. Etcd uses `fdatasync` to synchronize each write it performs in the WAL and sequential 500 IOPS of 8000 bytes for heavily loaded clusters at 2ms. A benchmarking tool can be used to obtain these metrics.
```

Comment:
Is this right? 

It is recommended to run etcd on a block device able to write at least 50 IOPS of 8000 bytes in lengthy sequentially

(the lengthy sequentially bit"